prime-router/settings/organizations.yml

```diff
@@ -1277,4 +1277,134 @@
         host: sftp
         port: 22
         filePath: ./upload
-        credentialName: DEFAULT-SFTP
\ No newline at end of file
+        credentialName: DEFAULT-SFTP
```

Comment:
We don't need to add any setting to organizatios.yml any more.  Please move the setting to setting/STLTs/VT/vt-doh.yml